test/ui/test_popup_plot_1d.py

```diff
@@ -0,0 +1,65 @@
+# package imports
+from RefRed.plot.popup_plot_1d import PopupPlot1d
+
+# third party imports
+import pytest
+from qtpy import QtCore
+
+
+def test_popup_plot_1d(qtbot, main_gui):
+    window_main = main_gui(show=True)
+
+    # Using qtbot.mouseClick twice to emulate a double bring is not instantiating up the popup
+    # q_rectangle = yi_plot.contentsRect()
+    # qtbot.mouseClick(yi_plot, QtCore.Qt.LeftButton, pos=q_rectangle.center())
+    # qtbot.mouseClick(yi_plot, QtCore.Qt.LeftButton, pos=q_rectangle.center())
```

Comment:
Do we still need these lines?

Reply:
I wanted to leave them for documentation. It's puzzling that they didn't work.